### PR TITLE
Fix azure terraform

### PR DESCRIPTION
**Pull request by kron4eg**
**What this PR does / why we need it**:
Fresh versions of azurerm provider require those changes, so keeping up to date config.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
keep azure terraform config up2date
```

### CODE REVIEW

File: examples/terraform/azure/output.tf [hunk 1]
```diff
@@ -47,7 +47,7 @@ output "kubeone_workers" {
     # following outputs will be parsed by kubeone and automatically merged into
     # corresponding (by name) worker definition
     "${var.cluster_name}-pool1" = {
-      replicas = 1
+      replicas = 0
```
Comment (xmudrii) [inline examples/terraform/azure/output.tf:50]: Why we reduced replicas to 0?

Comment (kron4eg) [reply examples/terraform/azure/output.tf]: slipped by error

**Review by xmudrii — APPROVED** [review]
/lgtm

**Pull request merged**